packages/components/music/UploadMusicModal/UploadTrack.tsx

```diff
@@ -52,7 +55,12 @@ interface Props {
 const UPLOAD_ALBUM_MODAL_WIDTH = 564;
 
 export const UploadTrack: React.FC<Props> = ({ onUploadDone }) => {
-  const { setToastError } = useFeedbacks();
+  const [isMapShown, setIsMapShown] = useState(false);
+  const [location, setLocation] = useState<
+    CustomLatLngExpression | undefined
```

Comment:
You dont need to precisely define type undefined. If default value is empty then the type is automatically including `undefined`

Reply:
Yes, I don't know why I did that